### PR TITLE
Add a new horizontal layout for the report

### LINKED ISSUE #50: Add a new horizontal layout for the report
Hi, I am new to mahjong and find akochan-reviewer is useful for learning. Thanks a lot for this project!

However, I found it is a litter hard to check the review result and the tenhou replaying at the same time. There is only limited space if I don't collapse the tenhou replay iframe.

I try to add a horizontal layout to place the tenhou replay to the left and the review result to the right side. Here are my changes https://github.com/JaySon-Huang/akochan-reviewer/tree/hori_style

I am thinking of merging it back to this repo. Would you mind accepting the layout change?

A preview for the report with horizontal layout:
[chankan.json.html.zip](https://github.com/Equim-chan/akochan-reviewer/files/8308615/chankan.json.html.zip)
image

### PR DESCRIPTION
**Pull request by JaySon-Huang**
close #50 

As mentioned in #50, I find there is only limited space if I don't collapse the tenhou replay iframe.

I add a horizontal layout to place the tenhou replay to the left and the review result to the right side.

A preview for the report with horizontal layout:
[chankan.json.html.zip](https://github.com/Equim-chan/akochan-reviewer/files/8308615/chankan.json.html.zip)
image

### CODE REVIEW

File: templates/report.html [hunk 1]
```diff
@@ -14,9 +14,12 @@
   <title>{% if lang == "en" %}Replay Examination{% else %}牌譜検討{% endif %}</title>
 </head>
 
-<body>
+<body {% if layout == "horizontal" %} horizontal="1" {% endif %} >
   <h1>{% if lang == "en" %}Replay Examination{% else %}牌譜検討{% endif %}</h1>
 
+  <input type="radio" name="style" id="radio_style_v" {% if layout == "vertical" %} checked="true" {% endif %} onclick="toggleStyle()" />vertical
+  <input type="radio" name="style" id="radio_style_h" {% if layout == "horizontal" %} checked="true" {% endif %} onclick="toggleStyle()" />horizontal
```
Comment (Equim-chan) [inline templates/report.html:21]: Wrapping with a `label` tag is better, like `<label><input ...>vertical</label>`.

File: templates/report.html [hunk 2]
```diff
@@ -252,9 +267,35 @@ <h1 id="kyoku-{{ item.kyoku }}-{{ item.honba }}" class="kyoku-heading">
...
+        for (var i = 0; i < lboxes.length; i++) {
+          lboxes[i].setAttribute("horizontal", "1")
+        }
+        rboxes = document.querySelectorAll(".r-box")
+        for (var i = 0; i < rboxes.length; i++) {
+          rboxes[i].setAttribute("horizontal", "1")
+        }
+      }
+    }
+  </script>
```
Comment (Equim-chan) [inline templates/report.html:298]: Maybe create a separate standalone javascript template file and `include` it here? Just like the CSS.

Comment (JaySon-Huang) [reply templates/report.html]: done

File: templates/report.html (hunk 1 above)
Comment (JaySon-Huang) [reply templates/report.html]: done

File: templates/report.html [hunk 3]
```diff
@@ -14,9 +14,12 @@
   <title>{% if lang == "en" %}Replay Examination{% else %}牌譜検討{% endif %}</title>
 </head>
 
-<body>
+<body {% if layout == "horizontal" %} horizontal="1" {% endif %} >
   <h1>{% if lang == "en" %}Replay Examination{% else %}牌譜検討{% endif %}</h1>
 
+  <label for="radio_style_v"><input type="radio" name="style" id="radio_style_v" {% if layout == "vertical" %} checked="true" {% endif %} onclick="toggleStyle()" />vertical</label>
+  <label for="radio_style_v"><input type="radio" name="style" id="radio_style_h" {% if layout == "horizontal" %} checked="true" {% endif %} onclick="toggleStyle()" />horizontal</label>
```
Comment (JaySon-Huang) [inline templates/report.html:21]:
```suggestion
  <label for="radio_style_h"><input type="radio" name="style" id="radio_style_h" {% if layout == "horizontal" %} checked="true" {% endif %} onclick="toggleStyle()" />horizontal</label>
```

File: templates/report.html [hunk 4]
```diff
@@ -18,7 +18,7 @@
   <h1>{% if lang == "en" %}Replay Examination{% else %}牌譜検討{% endif %}</h1>
 
   <label for="radio_style_v"><input type="radio" name="style" id="radio_style_v" {% if layout == "vertical" %} checked="true" {% endif %} onclick="toggleStyle()" />vertical</label>
-  <label for="radio_style_v"><input type="radio" name="style" id="radio_style_h" {% if layout == "horizontal" %} checked="true" {% endif %} onclick="toggleStyle()" />horizontal</label>
+  <label for="radio_style_h"><input type="radio" name="style" id="radio_style_h" {% if layout == "horizontal" %} checked="true" {% endif %} onclick="toggleStyle()" />horizontal</label>
```
Comment (Equim-chan) [inline templates/report.html:21]: Maybe the `for` is not needed at all?

**Pull request merged**